src/ChildInterface.php

```diff
@@ -8,5 +8,5 @@
 
 interface ChildInterface
 {
-    public static function create(Messenger $messenger, LoopInterface $loop): void;
+    public static function create(Messenger $messenger, LoopInterface $loop);
```

Comment:
That `void`  return type hint isn't a bug, and intentional as well. However, if I can get the `react/filesystem` package to work with it removed. I'm accepting that for now and tag `4.0.0` on this package to get it unblocked. Will then do a `5.0.0` later on with additional changes I've had in mind.